.gitpod.yml

```diff
@@ -0,0 +1,31 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
should we remove this since we changed the gitpod file.

Reply:
lol yea

Reply:
updated